.gommit.toml

```diff
@@ -0,0 +1,18 @@
+[config]
+exclude-merge-commits=true
+check-summary-length=false
+
+[matchers]
+all="(?:feat|test|fix|refactor)\\(\d*?\\): .*?\n(?:\n?(?:\\* |  ).*?\n)*"
```

Comment:
What's the goal of this regex? I can't understand it

Reply:
Removed.

It was for commit message checking - I failed to remove it when I removed the circle CI config element for commit checking.